rest-api/tools/import_organizations.py

```diff
@@ -108,6 +108,41 @@ def _insert_entity(self, entity, existing_map, session, dry_run):
     super(HPOImporter, self)._insert_entity(entity, existing_map, session, dry_run)
 
 
+  def delete_sql_statement(self, session, hpo_id_list):
+    str_list = ','.join([str(i) for i in hpo_id_list])
+    sql = """
+          DELETE FROM hpo
+          WHERE hpo_id IN ({str_list})
+          AND NOT EXISTS(
+          SELECT * FROM participant WHERE hpo_id in ({str_list}))
+          AND NOT EXISTS(
+          SELECT * FROM participant_history WHERE hpo_id IN ({str_list}))
+          AND NOT EXISTS(
+          SELECT * FROM participant_summary WHERE hpo_id IN ({str_list}))
+          AND NOT EXISTS(
+          SELECT * FROM organization WHERE hpo_id IN ({str_list}))
+          AND NOT EXISTS(
+          SELECT * FROM site WHERE hpo_id IN ({str_list}))
+          """.format(str_list=str_list)
+
+    session.execute(sql)
+
+  def _cleanup_old_entities(self, session, row_list):
+    self.hpo_dao = HPODao()
+    existing_hpos = set(hpo.name for hpo in self.hpo_dao.get_all())
+    hpo_group_list_from_sheet = [row[HPO_AWARDEE_ID_COLUMN].upper() for row in row_list]
+
+    hpos_to_remove = existing_hpos - set(hpo_group_list_from_sheet)
+    if hpos_to_remove:
+      hpo_id_list = []
+      for hpo in hpos_to_remove:
+        old_hpo = self.hpo_dao.get_by_name(hpo)
```

Comment:
Add logging here for the awardee being deleted?

Reply:
done. Got a little over zealous removing the debugging logging.